### PR TITLE
adding 2023 year in review, adding blog, updating Mkdocs Material

### PR DESCRIPTION
**Pull request by MasonEgger**

### CODE REVIEW

File: foundation/docs/blog/posts/2023-review.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+For nearly the entirety of its lifetime, the conference was the Foundation's
+only expense and fundraising event via sponsorships. With the expansion into 
+meetups and the eventual goal of supporting local meetups we have to rethink our 
+fundraising efforts. We hope to be able to roll out a sponsorship package for
+sponsoring the PyTexas Meetup and even the Foundation overall within this coming
+year, so keep an eye on our website and socials for its announcement.
+
+#### **Increase Awareness Through Marketing**
+This mostly means more social media awareness. We're going to experiment with
```
Comment (thejcannon) [inline foundation/docs/blog/posts/2023-review.md:115]: Did you want a newline here? Running this through `prettier` might fixup all the little things

**Review by thejcannon — APPROVED** [review]
🎉

File: foundation/docs/blog/posts/2023-review.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+limitation on message retention we decided the move was necessary. While we
+made the move in 2022, it wasn't until 2023 that we started to see people actually
+engaging in the server. We now host our monthly meetup there (more below), a weekly
+Friday hangout, and have various text channels for discussion. 
+
+[Join the PyTexas Discord](https://discord.gg/jNPAbcNukj){ .md-button .md-button--primary }
+
+### PyTexas Meetup
+
+After various discussions with Texas Pythonista's we discovered that Texas was missing
```
Comment (toonarmycaptain) [inline foundation/docs/blog/posts/2023-review.md]: Should the apostrophe come after the s: `Pythonistas'`?

**Review by toonarmycaptain — APPROVED** [review]

File: foundation/docs/blog/posts/2023-review.md (hunk 2 above)
Comment (thejcannon) [reply foundation/docs/blog/posts/2023-review.md]: Wait would there be an apostrophe at all? Isn't it just plural?

Comment (toonarmycaptain) [reply foundation/docs/blog/posts/2023-review.md]: Correct. 🤦🏼

Comment (MasonEgger) [inline foundation/docs/blog/posts/2023-review.md]:
```suggestion
After various discussions with Texas Pythonistas we discovered that Texas was missing
```

Comment (MasonEgger) [reply foundation/docs/blog/posts/2023-review.md]: Fixed

**Pull request merged**